lib/gui/components/modal/services/modal.js

```diff
@@ -56,18 +56,32 @@ module.exports = function($uibModal, $q) {
       close: modal.close,
       result: $q((resolve, reject) => {
         modal.result
-          .then(resolve)
-          .catch((error) => {
+          .then((value) => {
```

Comment:
Can we put the `.then()` on the same line as the promise, to avoid the indentation to the whole section below? For example:

```js
modal.result.then((value) => {
  AnalyticsService.logEvent('Modal closed', {
    method: 'User accepted',
    value: value
  });

  resolve(value);
}).catch((error) => {
  // Bootstrap doesn't 'resolve' these but cancels the dialog;
  // therefore call 'resolve' here applied to 'false'.
  if (error === 'escape key press' || error === 'backdrop click') {
    AnalyticsService.logEvent('Modal closed', {
      method: error
    });

    resolve();

  // For some annoying reason, UI Bootstrap Modal rejects
  // the result reason if the user clicks on the backdrop
  // (e.g: the area surrounding the modal).
  } else {
    AnalyticsService.logEvent('Modal closed', {
      method: 'rejected'
    });

    return reject(error);
  }

});
```